### ISSUE TITLE
Monthly mycetoma meeting 6th July 2023

### ISSUE THREAD

**Issue by mattodd** [OpenSourceMycetoma/Monthly-zoom-meetings#7]
Time: July 6th (12 noon UK, 1pm EU, 9pm Aus) [Timezones](https://www.timeanddate.com/worldclock/meetingdetails.html?year=2023&month=7&day=6&hour=11&min=0&sec=0&p1=43&p2=136&p3=37&p4=771&p5=235&p6=152&p7=770&p8=250&p9=152&p10=240).

The Outlook invite can be found [here](https://github.com/OpenSourceMycetoma/Series-1-Fenarimols/files/9674081/Monthly.MycetOS.meeting00.zip)

Location: https://uni-sydney.zoom.us/j/99281549497. In case of failure, use https://ucl.zoom.us/j/99789829423 and in case of double failure use [MycetOS meeting teams link](https://teams.microsoft.com/l/meetup-join/19%3ameeting_M2I2MDcxZTgtYjZjOS00YTdmLTgwZTMtNWQ3OTc4NDY0MDNj%40thread.v2/0?context=%7b%22Tid%22%3a%22526638ba-6af3-4b0f-a532-a1a511f4ac80%22%2c%22Oid%22%3a%22a5ed84b8-5af1-42c6-9f8d-240f23a5d997%22%7d)
Chair: ErasmusMC
Minute taker: USyd
Previous meeting: #6 
Recording: https://youtu.be/BM88SKPsLdM
Present: @mattodd @wwjvdsande @dmitrij176 @bebi78 @MA-Jjingyi @MFernflower @KlementineJBS 
Apologies: none

AGENDA

### Series 1 - Fenarimols

_Final Molecules/SAR Gaps_

[@fantasy121 completed the selection of Epichem molecules](https://github.com/OpenSourceMycetoma/Monthly-zoom-meetings/issues/5#issuecomment-1530752314), and the next step is to get them to Wendy.

@fantasy121 shared [the list of compounds that have been shipped from Sydney to ErasmusMC](https://github.com/OpenSourceMycetoma/Monthly-zoom-meetings/issues/5#issuecomment-1542026215). 

- [ ] @MA-Jjingyi to evaluate these latest fenarimols and post data to the Master List.

- [ ] @mattodd to email the finalised list to Epichem (was there a query about whether the relationship between Epichem and DNDi still existed?)

- [x] @meh-cyprian to proceed to try to model fenarimol compounds in potential protein target [here](https://github.com/OpenSourceMycetoma/Series-1-Fenarimols/issues/68#issuecomment-1546869698), or to clarify what more data are needed. Aim is to better understand the SAR via docking vs homology model of protein.

_Paper:_

- [ ] @dmitrij176 to upload offline copy of fenarimol lab notebook to UCL respository and provide link (@fantasy121 is https://doi.org/10.25910/CS81-G976). @mattodd to assist with copying data over.

- [x] @dmitrij176 to upload his Biological supplemental data (not in Labarchives). Link to the data: https://github.com/OpenSourceMycetoma/Monthly-zoom-meetings/issues/6#issuecomment-1559101604

- [ ] @mattodd to review final version of manuscript. Please don't go through the manuscript till @mattodd is finished. After this is done, @MA-Jjingyi and @wwjvdsande will generate an Excel sheet from our current Master List displaying all data on the fenarimols tested. In meeting: @mattodd will forward edited version of manuscript to @dmitrij176 and @fantasy121 for completion.

It was decided at the previous meeting #6 that abbreviated MYOS codes would be used for the compounds, to maintain the link between compounds in the paper and compounds in the live project. The codes needed are just the initial integers, not the batch/salt codes etc. Can also remove zeroes for readability, so e.g. MYOS7.

The cut-off for in vivo data is end of larva season 2022: In vitro data will continue to be collected in 2023 (In vivo data 2023 season might be added to later revisions of manuscript but _don’t wait for this before submission_)

_Siderophores:_

New transcriptomic data from @wwjvdsande in the galleria grain model showed that siderophores are produced (https://github.com/OpenSourceMycetoma/What-other-molecules-to-screen/issues/10).. These siderophores might be used as a trojan horse to get molecules into the grain. Could we couple fenarimols to fungal-relevant siderophores may be to get them in? The idea of the siderophores was of interest and all partners agreed to consider whether this could be done at the different sites. There is also potential relevance to other series. 

- [ ] All to report back on potential structures (in the original issue https://github.com/OpenSourceMycetoma/What-other-molecules-to-screen/issues/10) and ways they might be produced within the resources available.
- [ ] @wwjvdsande will pursue a link with Doyle in Ireland who might be able to attach a fluorophore to a siderophore that would permit assessment of whether the siderophore transported the fluorophore into the grain.

### Series 2 - Aminothiazoles

[Meeting between the chemistry team and the high schools took place](https://github.com/OpenSourceMycetoma/Series-2-Aminothiazoles/issues/14#issuecomment-1523301479). The contributors now all have their own targets. @KlementineJBS will be the corresponding researcher for this project at USYD (Replacement for @kym834). UCL Masters student cohort begins in late May and will report back.

- [ ] @mattodd to organise next meeting of this group needs to be organised, e.g. July?
- [ ] @dmitrij176 to ship best Whitgift sample received to date in next shipment to @wwjvdsande
- [ ] @dmitrij176 to generate MyOS code for the sample and add to the Master List.
- [x] Whitgift school need help with ordering a chemical: 2-bromo-1-(pyridin-2-yl)ethan-1-one HBr [this?](https://oakwoodchemical.com/ProductsList.aspx?CategoryID=-2&txtSearch=2964). They tried Apollo Scientific and are trying to sort out paperwork for Oakwood, but they are having trouble ordering. They need 1g per group, so 10 g total. They need help with ordering it. <-- problem resolved
- [ ] Was there an action item on @dosreic re trying to secure samples related to this paper https://github.com/OpenSourceMycetoma/Series-2-Aminothiazoles/issues/11 ?
- [ ] @KlementineJBS to consider an edit to the Master List and report back: Generally we are not doing well at keeping track of who is making what. We need contributors to clarify their targets, and these should be included in the Master List as targets, where the "target" tag can be removed upon synthesis. We need a commitment from all contributors to help with data management on the Master List. Volunteer needed to ensure Master List has the capability to add the tag "target" to clarify which molecules are suggestions, ideally with a link to the URL where the molecule is suggested.

### Series 5 - Benzimidazoles

Oxazoles:
Good potencies observed for recent compound batches (e.g. https://github.com/OpenSourceMycetoma/Series-1-Fenarimols/issues/82#issuecomment-1325150305). Promising because they resembled the benzimidazoles which showed potent in vivo efficacy. Some oxazoles will be tested against Falciformispora senegalensis too. This series will be evaluated in vivo in the new larval season.

- [ ] @dosreic to try to obtain samples.
- [ ] @alintheopen and @memm0740 would see if they can access them too.

@AndreaOtago has generated [a new set of benzoxazoles](https://github.com/OpenSourceMycetoma/series-5-benzimidazoles/issues/3#issuecomment-1556652617) and these will be shipped to @wwjvdsande 

- [ ]  @fantasy121 to ask the two new Sydney Hons students (Peiyan and Marita) to join Github and share their IDs (Marita has joined and is @memm0740.
- [ ] Peiyan and @memm0740 to do the Scifinder lit search of the surrounding chemical space: are there other groups making similar compounds whom we could ask for donations?
- [x] Marita to post her synthesis plan for series 5 that was presented last month.

### Series 6: ketoximes

[ ] @dmitrij176 to share the figure of the compounds that have been shipped to ErasmusMC for testing. This can be found at https://github.com/OpenSourceMycetoma/Series-6-Ketoximes/issues/5. @dmitrij176 provided an update in the meeting on the status of the latest compounds.

### Misc

DMPK measurement offer from the Wellcome Centre Anti-Infectives Research (WCAIR) at the University of Dundee, which often hosts trainees from around the world and who could run some 1st tier in vitro DMPK assays for MycetOS compounds as part of the training set for these folks. They have one trainee coming this July for 3 months and they could start doing this, if of interest. A meeting was held on June 16th to discuss. @bebi78 has made suggestions at https://github.com/OpenSourceMycetoma/Monthly-zoom-meetings/issues/6#issuecomment-1600655716, and some earlier comments were at https://github.com/OpenSourceMycetoma/Monthly-zoom-meetings/issues/5#issuecomment-1538392028.

- [ ] All to complete this list of compounds to send to Dundee for DMPK measurements by what deadline?
- [ ] @mattodd to set up new issue to finalise list.

MMV Global Health Priority Box screened against Madurella mycetomatis and Falciformispora senegalensis. The data are uploaded in the Master List. Are there any standout hits - yes! MMV1593278, MMV1804559 and MMV020335. Structures are at https://github.com/OpenSourceMycetoma/Monthly-zoom-meetings/issues/6#issuecomment-1567312472. In meeting it was mentioned that the next stage is to evaluate these compounds in the in vivo model.

Rohan’s contributed compounds: 96 well plate received. To be screened. Data for these compounds [here](https://github.com/OpenSourceMycetoma/Series-1-Fenarimols/issues/84#issuecomment-1507518700).

- [ ] **Need a student volunteer** to enter the data in Rohan's spreadsheet into the Master List. We have no [MyOS codes](https://github.com/OpenSourceMycetoma/MycetOS-Tech-Ops/wiki/The-Molecule-Numbering-Convention) for them.

Kinase inhibitor library. The updated KCGS library was ordered on 1st of May. It has arrived in Erasmus MC. Bio-informatical analyses have been carried out by Mickey. @bebi78 can send some 10 targets. Action on this?

Bioinformatics: Waleed requested information on which aspect of bioinformatics should be prioritised to work on first. @wwjvdsande corresponded offline with him. This will include a Phylogenetic tree to work out relationships of various species were identified as high priorities so that we can search the literature for closely related species to Mycetoma, and for the potential structures of the drug target, which would enable us to simulate our data/estimation/computational models. In the May meeting Tariq Ayub attended from the Qasim group (need Github userID!) who will advise on what can be done in terms of future contributions.

**AOB**
Other Updates:
USYD
UCL
Erasmus
Other sites

**Comment by memm0740** [comment]
Unfortunately won't be attending the meeting today. I have created some slides on where I am at so far in synthesising the benzimidazole compounds:
[MycetOS July meeting.pptx](https://github.com/OpenSourceMycetoma/Monthly-zoom-meetings/files/11963226/MycetOS.July.meeting.pptx)

I am still work on synthesising the lead compound:
![image](https://github.com/OpenSourceMycetoma/Monthly-zoom-meetings/assets/124969922/d5427647-cb28-4dfe-a354-cb3653d65c7b)

But I hope to start on some SAR very soon !

**Comment by MA-Jjingyi** [comment]
Unluckily, the Italy factory produced larvae is going through flood. then we tested larvae from 2 other brands. we only can start treatment after the quality of larvae must be verified. In the meantime, we hope the factory from Italy will be reproduced larvae soon. then we can work as usual.

**Comment by mattodd** [comment]
Hi @memm0740 - that's great. Can you please share a link to the ELN? Important that raw data are available for each project. We can also add to the wiki under "Sources of Data".

**Comment by wwjvdsande** [comment]
For those who are late, join us in:  https://ucl.zoom.us/j/99789829423

**Comment by MFernflower** [comment]
have any novel phenothiazines been made? @bebi78 

if not - since many phenothiazines are well known - series 3 should NOT be sent for DMPK

**Comment by bebi78** [comment]
@MFernflower No, but maybe @rohandavis has some suggestions from his compound list.

**Comment by kym834** [comment]
@dmitrij176 the whitgift compound is one we've already made so it doesn't need to be assigned a MYOS number. They've made a repeat in the first instance to get their head around the synthesis and make sure everything works. 

@mattodd - the compound Sevenoaks was working on, has been synthesised and analysed? Please tag/share the schools meeting dets with me once you've organised.

**Comment by KlementineJBS** [comment]
Chair: UCL
Minute taker: USyd
Previous meeting: #6 
Recording: [here]
Present: Mat Todd, Jingyi Ma, Dmitrij Melechov, Wendy van de Sande, Bernhard, Tony
Apologies: Hung Phat Duong, Marita Emmanuel

## MINUTES

### Series 1 - Fenarimols

_Final Molecules/SAR Gaps_
- [ ] @MA-Jjingyi to evaluate these latest fenarimols and post data to the Master List.
- _Ma has not been able to test latest fenarimols yet due to issues in obtaining larvae. Will be done as soon as these issues can be resolved._
- [ ] @mattodd to email the finalised list to Epichem (was there a query about whether the relationship between Epichem and DNDi still existed?)
- _Still to be done_
- [ ] @meh-cyprian to proceed to try to model fenarimol compounds in potential protein target [here](https://github.com/OpenSourceMycetoma/Series-1-Fenarimols/issues/68#issuecomment-1546869698), or to clarify what more data are needed. Aim is to better understand the SAR via docking vs homology model of protein.
Someone from this lab will join Mat’s lab in September – hopefully this makes it easier to get this done. 
_Paper:_

- [ ] @dmitrij176 to upload offline copy of fenarimol lab notebook (all relevant experiments) to UCL respository and provide link (@fantasy121 is https://doi.org/10.25910/CS81-G976). @mattodd to assist with copying data over.
- _Mat will try to sort this out._
- [x] @mattodd to review final version of manuscript. Please don't go through the manuscript till @mattodd is finished. After this is done, @MA-Jjingyi and @wwjvdsande will generate an Excel sheet from our current Master List displaying all data on the fenarimols tested.
- _Mat has made various changes including to SAR and will send it back to Hung and Dmitrij to review changes and finish experimental. Also need to make it clearer – how do these results fit into existing knowledge? This will probably require a literature search. 
- [ ] Compounds in paper need to be numbered with abbreviated MYOS codes e.g. MYOS7. 
The cut-off for in vivo data is end of larva season 2022: In vitro data will continue to be collected in 2023 (In vivo data 2023 season might be added to later revisions of manuscript but _don’t wait for this before submission_)

#### Siderophores
New transcriptomic data from @wwjvdsande in the galleria grain model showed that siderophores are produced (https://github.com/OpenSourceMycetoma/What-other-molecules-to-screen/issues/10). These siderophores might be used as a trojan horse to get molecules into the grain. Could we couple fenarimols to fungal-relevant siderophores may be to get them in? The idea of the siderophores was of interest and all partners agreed to consider whether this could be done at the different sites. There is also potential relevance to other series. 
We have previously discussed what molecules could be attached to siderophores to explore this idea – everyone is encouraged to share ideas in the original issue. (Please post a link if you have it!)

- [ ] All to report back on potential structures (in the original issue OpenSourceMycetoma/What-other-molecules-to-screen#10) and ways they might be produced within the resources available.

Wendy has been talking to Professor Doyle about using his fluorescent-labelled siderophores to show that siderophores can indeed enter the grain – i.e. proof of concept. If this goes well we can move towards coupling molecules of interest onto the siderophores. Details still to be discussed with Doyle. 
Mat is hopeful that if UCL can get a MALDI, it will allow us to visualise where molecules are located within the grain without needing to use fluorescent labelling.

### Series 2 - Aminothiazoles
UCL students have started making molecules – some will be ready this month, others later in the summer. This is organised by Erica. 
- [ ] @mattodd to organise next meeting of this group needs to be organised, e.g. July? 
- [ ] @dmitrij176 to ship best Whitgift sample received to date in next shipment to @wwjvdsande 
- [ ] @dmitrij176 to generate MyOS code for the sample and add to the Master List.
@KlementineJBS has updated the [Wiki]( https://github.com/OpenSourceMycetoma/Series-2-Aminothiazoles/wiki/All-Compounds-Made-To-Date) with the compounds successfully synthesised by USyd in June.
- [ ] @KlementineJBS to add new target molecules to appropriate page in wiki
- [ ] @KlementineJBS to try and edit the master list to include a “target”/”in synthesis” tag with URL linking back to where it was originally suggested 
- Was there an action item on @dosreic re trying to secure samples related to this paper https://github.com/OpenSourceMycetoma/Series-2-Aminothiazoles/issues/11 ?
Waiting on the DNDI for this.

### Series 5 - Benzimidazoles

Oxazoles:
Good potencies observed for recent compound batches (e.g. https://github.com/OpenSourceMycetoma/Series-1-Fenarimols/issues/82#issuecomment-1325150305). Promising because they resembled the benzimidazoles which showed potent in vivo efficacy. Some oxazoles will be tested against Falciformispora senegalensis too. This series will be evaluated in vivo in the new larval season.
¬
- [ ] @dosreic to try to obtain samples.
Unsure exactly what samples are to be obtained – ones similar to existing compounds?
- [ ] @alintheopen and @memm0740 would see if they can access them too.
- [ ]  @KlementineJBS to share Peyian’s GitHub username (Marita has joined as @memm0740)
- [ ] Peiyan and @memm0740 to do the Scifinder lit search of the surrounding chemical space: are there other groups making similar compounds whom we could ask for donations?

### Series 6: ketoximes

Dmitrij has shared the figure of the compounds that have been shipped to ErasmusMC for testing. This can be found at https://github.com/OpenSourceMycetoma/Series-6-Ketoximes/issues/5.



### Misc

DMPK measurement offer from the Wellcome Centre Anti-Infectives Research (WCAIR) at the University of Dundee, which often hosts trainees from around the world and who could run some 1st tier in vitro DMPK assays for MycetOS compounds as part of the training set for these folks. They have one trainee coming this July for 3 months and they could start doing this, if of interest. A meeting was held on June 16th to discuss. @bebi78 has made suggestions at https://github.com/OpenSourceMycetoma/Monthly-zoom-meetings/issues/6#issuecomment-1600655716, and some earlier comments were at https://github.com/OpenSourceMycetoma/Monthly-zoom-meetings/issues/5#issuecomment-1538392028.
Need 2-3 mg of highly pure compound – we will send 30-35 molecules, about 5 from each series. Ideally should send highly potent compounds, but also would be good to include slightly less potent compounds if they have higher predicted solubility. Ideally, the selection should be structurally diverse. Separate enantiomers should only be sent if they have highly different potencies. Everyone should discuss this offline and determine a list of compounds. 
Samples need to be ready/sent off by end of the month.
- [ ] @Mattodd will look into series 1 
- [ ] @KlementineJBS will look into series 2
- [ ] Need to check whether MMV will authorise us to send promising compounds from MMV (MMV1593278, MMV1804559 and MMV020335)  – structures available [here]( https://github.com/OpenSourceMycetoma/Monthly-zoom-meetings/issues/6#issuecomment-1567312472). Alternatively, carbendazim would be interesting to look at. 
- [ ] @dmitrij176 will look into/confirm selection for series 6 

Everyone should add their suggested selection to this issue (insert link). Make sure to provide compound name, MYOS code, structure, SMILES and the reasoning behind the selection. 

- [ ] All to complete this list of compounds to send to Dundee for DMPK measurements by what deadline?

- Rohan’s contributed compounds: 96 well plate received. To be screened. Data for these compounds [here](https://github.com/OpenSourceMycetoma/Series-1-Fenarimols/issues/84#issuecomment-1507518700).

-	[ ] @KlementineJBS to ask @memm07040 to enter data from Rohan’s spreadsheet into Master list and provide MYOS codes.

Kinase inhibitor library. The updated KCGS library was ordered on 1st of May and has been shipped. Bio-informatical analyses have been carried out by Mickey. @bebi78 can send some 10 targets. Action on this?

-	[ ] @MA-Jjingyi can potentially look into testing whether kinases are targeted if/when she has time.
Mickey will be looking into this soon and has already done bioinformatics work to determine what kinases are present in the genome. Likely to be screen in early autumn.

Bioinformatics: Waleed requested information on which aspect of bioinformatics should be prioritised to work on first. @wwjvdsande corresponded offline with him. This will include a Phylogenetic tree to work out relationships of various species were identified as high priorities so that we can search the literature for closely related species to Mycetoma, and for the potential structures of the drug target, which would enable us to simulate our data/estimation/computational models. In the May meeting Tariq Ayub attended from the Qasim group (**need Github userID!**) who will advise on what can be done in terms of future contributions.
-	[ ] Bernhard to invite Tariq to meetings and ask him to make a GitHub account so he can stay in the loop

**AOB**

UCL
Dmitrij has obtained a crystal structure for one of the Series 6 compounds. He’s also successfully separated the enantiomers of the positive control – this could be an interesting choice for DMPK studies. Need to see if the two enantiomers have different potency first. 
Would also be interesting to test different salts – Dmitrij can perhaps choose a potent analogue and try forming the salt.

**Comment by KlementineJBS** [comment]
Hi - we have a meeting scheduled today/tonight in the calendar but I remember we discussed maybe not having it today as it's so soon after our last meeting. What did the decision end up being?

**Comment by mattodd** [comment]
Hi @KlementineJBS @animarfavi OK, well in that case given that @wwjvdsande and @csuth22 and @AndreaOtago are also apologies, and given that we only really have one issue to discuss - which is the compound shipment to Dundee - we can skip the in-person meeting today and instead deal with things offline. I will post an updated agenda for the next meeting (Aug 29th - chair USyd) and I'll use the time to collate the various decisions on shipping in #8. Ping @OpenSourceMycetoma/corecontrib @dosreic

**Comment by meh-cyprian** [comment]
Hi @mattodd sorry I haven't been active, 
U have generated the homology model and need to go on with docking the Fenarimol compounds. To achieve this, I will need the following
- The number of compounds that were successfully tested on the target protein
- The range of activity (highest and lowest)
- Brief description of the assay.
Thanks

**Comment by mattodd** [comment]
Hi @meh-cyprian 

The number of compounds that were successfully tested on the target protein: zero. All compounds are tested phenotypically (vs whole organism). The protein target is assumed, hence in part the need for this analysis.

The range of activity (highest and lowest). All compounds can be found in the [Master List](http://tinyurl.com/MycetomaMols). Filter by "Series 1". You only want in vitro potencies, and you'd need to select a consistent number, e.g. IC50 or IC90. Contact @MA-Jjingyi for any further clarification. @wwjvdsande may also have a view.

Brief description of the assay. No protein binding or biochemical assay, only potency in vitro. Again, if you need details, @MA-Jjingyi can provide, but I suspect this is not super-relevant for you.

**Comment by mattodd** [comment]
Hi @meh-cyprian did you do anything with the Master List of compound data in the last couple of days by any chance? We noticed that a load of data was deleted. It's been restored, I think.

https://github.com/OpenSourceMycetoma/Monthly-zoom-meetings/issues/10#issuecomment-1666518911

If you had been editing the spreadsheet recently, let us know.

**Comment by meh-cyprian** [comment]
Hi @mattodd  
I didn't do anything with the Master list I just downloaded the list because the day i was given access to the list.
I have a copy offline
